src/components/SearchBox.vue

```diff
@@ -161,7 +161,12 @@ export default {
 
         // we return the local value if the input is focused to avoid
         // concurrent updates when typing
-        const { searchInput } = this.$refs;
+        const { searchInput } =
+          (this.$scopedSlots.default &&
+            this.$scopedSlots.default()[0] &&
+            this.$scopedSlots.default()[0].context.$refs) ||
```

Comment:
this is essentially assuming there's a ref inside the scoped slot with that same name, but it probably would be better to pass `searchInputRef` as a data to the scoped slot, so that it can be bound as expected (bonus that it would also work in vue 3 without conditionals)

Reply:
Pushed a new commit.